<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Book_Rank</class>
 <widget class="QMainWindow" name="Book_Rank">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Book_Rank</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>620</width>
      <height>400</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>按收藏数排序</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>621</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="testLayout2"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>按借阅数排序</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>621</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="testLayout1"/>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnLoadData">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>450</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>加载数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_return">
    <property name="geometry">
     <rect>
      <x>404</x>
      <y>452</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string> 返回</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
